docs/llm_tools.md

```diff
@@ -0,0 +1,204 @@
+---
+title: "LLM Tools"
+sidebar_label: "LLM Tools"
+---
+
+LLM tools extend the functionality of OpenAI Conversation, Google Generative AI, and possibly other future LLM integrations. You can define your own LLM tool that these integrations can use.
+
+There are 3 options to define your own tool.
+
+## Option 1. Intent
+
+Create and register your own intent handler. Intents are automatically registered as LLM tools. This method allows to use the same code for other integrations that support intents.
```

Comment:
Instead of defining these as 3 options, lets document the APIs and also just say that intents are automatically registered.


```suggestion
All [Intents](/docs/intent_index) are automatically registered as LLM tools. This page describes the APIs for registering additional tools.
```

Reply:
The tool registration docs can be omitted given we're reducing scope of the PR to only register intents.